module/VuFind/src/VuFind/Db/Table/Comments.php

```diff
@@ -174,4 +174,55 @@ public function getStatistics()
         $result = $statement->execute();
         return (array)$result->current();
     }
+
+    /**
+     * Get a paginated result of all comments
+     * made by the user
```

Comment:
Add punctuation (and collapse to one line) for consistency.

```suggestion
     * Get a paginated result of all comments made by the user.
```